JuiceMaker/JuiceMaker/Model/FruitStore.swift

```diff
@@ -5,38 +5,50 @@
 //
 
 class FruitStore {
-    private var stock: [Fruit: Int] = [.strawberry: 10, .banana: 10, .pineapple: 10, .kiwi: 10, .mango: 10]
+    private var stock = [Fruit: Int]()
     
-    private func hasFruit(menu: Menu, total: Int) -> Bool {
-        let necessaryFruit = menu.count(to: total)
+    init(defaultStock: Int) {
+        Fruit.allCases.forEach {
+            stock[$0] = defaultStock
+        }
+    }
```

Comment:
```suggestion
    init(need: Int = 10) {
        Fruit.allCases.forEach {
            stock[$0] = defaultStock
        }
    }
```
=> 저번에 말씀드렸던건 위처럼 Default Parameter Value 를 집어넣으라는 말씀이었습니다.
그럼 평소에는 FruitStore()로 생성하다가도 테스트할 때 임의의 need 값을 집어넣을 수 있습니다 :) 

[해당 페이지](https://docs.swift.org/swift-book/LanguageGuide/Functions.html)에서 Default Parameter Values 로 검색해서 어떤것인지 알아보세요!

Reply:
음 다시 보니 need라는 이름은 부적절하네요. amount?

Reply:
Defualt Parameter Value를 이용해 수정했습니다.
제이슨의 말을 이제 이해했네요 😆